nucleus/payara-modules/hazelcast-bootstrap/src/main/java/fish/payara/nucleus/hazelcast/admin/SetHazelcastConfiguration.java

```diff
@@ -201,6 +201,9 @@
     @Param(name = "kubernetesServiceName", optional = true, alias = "kubernetesservicename")
     private String kubernetesServiceName;
 
+    @Param(name = "enableAutoPortSelection", optional = true, defaultValue = "true")
+    private boolean enableAutoPortSelection;
```

Comment:
This could also be renamed to just be `autoIncrementPort` to make it sound less robotic.
Also auto-port _selection_ sounds scary, like it's just going to pick one at random :P

Reply:
I like that better as well, the `enableAutoPortSelection` came from the ticket but I assume it can treat it as a suggestion and use `autoIncrementPort` instead.